webanno-model/src/main/java/de/tudarmstadt/ukp/clarin/webanno/model/AnnotationFeature.java

```diff
@@ -83,6 +83,9 @@
 
     private boolean visible = true;
     
+    @Column(nullable = false)
```

Comment:
If the column is not nullable, then we have a problem because existing rows do not have this column yet. The application will crash.

Reply:
True, I changed it accordingly: a3fbec436ce8ee67d6643c7cf2a968dba25b48e7